be/db_test.go

```diff
@@ -497,6 +497,67 @@ func TestGetTopScores(t *testing.T) {
 	})
 }
 
+func TestGetStats(t *testing.T) {
+	it(func() {
+		testCases := []struct {
+			name string
+			id   string
+
+			expectResponse *StatsResponse
+			expectError    bool
+		}{
+			{
+				name: "Get stats success",
+				id:   "0x1234",
+
+				expectResponse: &StatsResponse{
+					Version:           "2.0",
+					Id:                "0x1234",
+					KitnsDaily:        10,
+					KitnsDisbursed:    1000,
+					KitnsRefDaily:     0.25,
+					KitnsRefDisbusded: 5.5,
+				},
+				expectError: false,
+			}, {
+				name: "Get stats error",
+				id:   "0x5678",
+
+				expectResponse: nil,
+				expectError:    true,
+			},
+		}
+
+		recordColumns := []string{
+			"kitns_daily",
+			"kitns_disbursed",
+			"kitns_ref_daily",
+			"kitns_ref_disbursed",
+		}
+		for _, testCase := range testCases {
+			setUp()
+			if testCase.expectError {
+				mock.ExpectQuery("SELECT kitns_daily, kitns_disbursed, kitns_ref_daily, kitns_ref_disbursed	FROM users WHERE id = (.+)").
```

Comment:
May be format it for readability with ``?

Reply:
It matches the regexp. So even a number of spaces might be critical there. When it's in one line it's easier to count them and tweak it if necessary.